frontend/src/employee-frontend/components/reports/PresenceReport.tsx

```diff
@@ -63,7 +63,7 @@ function Presences() {
             <ReportDownload
               data={rows.value.map((row) => ({
                 ...row,
-                date: row.date.format('dd/MM/yyyy'),
+                date: row.date.formatIso(),
```

Comment:
This is a breaking change 😅 but I think `dd.MM.yyyy` would be more appropriate here.

Reply:
Hah, nice catch. It was intentional and I used ISO because it was widely used in the reports and should be more compliant in CSV exports. What do you think?

Reply:
Consistency is always good 👍